mono/mini/mini-llvm.c

```diff
@@ -4676,6 +4676,56 @@ get_float_const (MonoCompile *cfg, float val)
 		return LLVMConstFPExt (LLVMConstReal (LLVMFloatType (), val), LLVMDoubleType ());
 }
 
+static void
+emit_safepoint_poll_jit (EmitContext *ctx)
+{
```

Comment:
Can't this be merged with the aot version ?

Reply:
Ok, will try